# Franka robot state

float32[16] O_T_EE
float32[7] tau_J
float32[7] dtau_J
float32[7] q
float32[7] dq